var/spack/repos/builtin/packages/exaworks/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Exaworks(BundlePackage):
+    '''
+    Virtual package for the Exaworks SDK packages.
+    '''
+
+    homepage = 'https://exaworks.org/'
+    maintainers = ['andre-merzky']
+
+    version('0.1.0')
+
+    depends_on('adlbx',                   type=('build', 'run'))
+    depends_on('turbine',                 type=('build', 'run'))
+    depends_on('mpix-launch-swift',       type=('build', 'run'))
+
+    depends_on('flux-core',               type=('build', 'run'))
+    depends_on('flux-sched',              type=('build', 'run'))
+
+    depends_on('py-parsl',                type=('build', 'run'))
+
+    depends_on('py-radical-gtod',         type=('build', 'run'))
+    depends_on('py-radical-utils',        type=('build', 'run'))
+    depends_on('py-radical-saga',         type=('build', 'run'))
+    depends_on('py-radical-pilot',        type=('build', 'run'))
+    depends_on('py-radical-entk',         type=('build', 'run'))
+
```

Comment:
flake is complaining about this blank line.

```
  var/spack/repos/builtin/packages/exaworks/package.py:33: [W391] blank line at end of file
  flake8 found errors
```

Once that is resolved, this looks good!

Reply:
Another repo, another set of rules :-)  Thanks for noting this, this is fixed now!